src/contexts/NetworkContext.js

```diff
@@ -0,0 +1,61 @@
+import React, { createContext, useState, useEffect, useContext } from 'react';
+import storage from '../services/storage';
+
+const NetworkConext = createContext();
```

Comment:
Oh, this is also missing a 't' I think. I was wondering how you did not get any errors! XD